src/Components/__tests__/ToolTipModal.spec.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import Modal from '@mui/material/Modal';
+import Backdrop from '@mui/material/Backdrop';
+import Button  from '@mui/material/Button';
+
+import ToolTipModal from '../ToolTipModal';
+
+describe("<ToolTipModal />", () => {
+    let wrapper;
+    beforeEach(() => {
+        wrapper = mount(<ToolTipModal text={"test"} />)
+    })
+
+    it("renders a modal", () => {
+        expect(wrapper.find(Modal)).toHaveLength(1);
+    });
+
+    it("renders a button", () => {
+        expect(wrapper.find(Button)).toHaveLength(1);
+    });
+
+    it("displays passed in props tooltip data on the button", () => {
+        expect(wrapper.find(Button).text()).toEqual("test");
+    })
+
+    it("simulates button opening and closing modal", () => {
```

Comment:
Let's  separate our tests by "category", for lack of a better word. For example, we can separate UI tests and behavior/functionality tests like so:
```
describe("ui", () => {
    it("should render a modal", () => {
        ...
    })

    it("should render a button", () => {
        ...
    })

    it("should render the correct button text", () => {
        ...
    })
})

describe("clicking on the button", () => {
    it("should close the modal", () => {
        ...
    })

    it("should open the modal", () => {
        ...
    })
})
```

The main idea here being: organizing our tests under specific `describe` statements, and having our tests read like English sentences in the output. "clicking on the button...should close the modal" etc.

Reply:
That makes sense. I will update it to reflect these ideas.